src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+
+>***Note***
+>If the import is performed using the AtomicMode "OBJECT", where the import will import any data without validation errors, the overall status will still be `ERROR` if any errors was found.
+
+***validationReport***
+
+The `validationReport` may include `errorReports` and `warningReports` if any errors or warnings were present during the import. When present, they provide a detailed list of any errors or warnings encountered.
+
+For example, a validation error while importing a `TRACKED_ENTITY`:
+```json
+{
+  "validationReport": {
+    "errorReports": [
+      {
+        "message": "Could not find TrackedEntityType: `Q9GufDoplCL`.",
+        "errorCode": "E1005",
+        "trackerType": "TRACKED_ENTITY",
+        "uid": "Kj6vYde4LHh"
+      },
+      ...
+    ],
+    "warningReports" : [ ... ]
+  }
+}
+```
+
+The report contains a message and a code describing the actual error (See the [error codes](#error-codes) section for more information about errors). Additionally, the report includes the `trackerType` and `uid` which aims to describe where in the data the error was found. In this case, there was a `TRACKED_ENTITY` with the uid `Kj6vYde4LHh` which had a reference to a tracked entity type which was not found.
+
+>***Note 1***
+>When referring to the `uid` of tracker objects, they are labeled as their object names in the payload. For example, the `uid` of a tracked entity would in the payload have the name "trackedEntity". The same goes for "enrollment", "event" and "relationship" for enrollments, events and relationships respectively.
+
+>***Note 2***
+>If no uid is provided in the payload, the import process will generate new uids. This means the error report might refer to a uid that does not exist in your payload.
+
+>***Note 3***
+>Errors are represents issues with the payload which the importer can not circumvent. Any errors will block that data from being imported. Warnings on the other hand are issues where it's safe to circumvent them, but the user should be made aware that it happened. Warnings will not block data from being imported.
+
+***stats***
+
+The status provides a quick overview of the import. After an import is completed, this will be the actual counts representing how much data was created, updated, deleted or ignored.
+
+Example:
+```json
+{
+  "stats": {
+    "created": 2,
+    "updated": 2,
+    "deleted": 1,
+    "ignored": 5,
+    "total": 10
+  }
+}
+```
+`created` refers to how many new objects which was created. In general, objects without an existing uid in the payload will be treated as a new object.
+
+`updated` refers to the number of objects updated. Objects with an existing uid in the payload will be treated as updating an existing object.
```

Comment:
That is not always true. Earlier in the docs, we said that the uids can be used for new objects as well?

Reply:
rewrote. Clarified to say if the UID is in the payload AND we find it in the DB. then its an update, otherwise, new